compiler/quilt/test/test_command.py

```diff
@@ -687,6 +700,82 @@ def test_audit_not_admin_package(self):
         with self.assertRaises(command.CommandException):
             command.audit('foo/bar')
 
+    @patch('quilt.tools.command._find_logged_in_team', lambda: None)
+    def test_access_list(self):
+        self.requests_mock.add(
+            responses.GET,
+            '%s/api/access/foo/bar' % command.get_registry_url(None),
+            status=201,
+            json={
+                'users': ['foo']
+            }
+        )
+        command.access_list('foo/bar')
```

Comment:
In general, when you test a successful call, don't just test that an exception is not thrown -- validate the data

Reply:
Please apply this feedback to this whole PR, don't worry about anything outside this PR for now